### ISSUE TITLE
AttributeError exception when trying to send a message

### ISSUE THREAD

**Issue by azmeuk** [horazont/aioxmpp#34]
Related to #16. Tested with aioxmpp 0.7.0 and python 3.5
With some code inspired from the examples, I tried to send a simple message:
```python
    @asyncio.coroutine
    def do_things(self):
        stop_event = self.make_sigint_event()
        msg = aioxmpp.stanza.Message(type_="normal")
        yield from self.client.stream.send_and_wait_for_sent(msg)
        yield from stop_event.wait()
```
```python
Traceback (most recent call last):
  File "/home/eloi/dev/push-server/local.virtualenv/lib/python3.5/site-packages/aioxmpp/stream.py", line 680, in _done_handler
    task.result()
  File "/usr/lib64/python3.5/asyncio/futures.py", line 274, in result
    raise self._exception
  File "/usr/lib64/python3.5/asyncio/tasks.py", line 239, in _step
    result = coro.send(None)
  File "/home/eloi/dev/push-server/local.virtualenv/lib/python3.5/site-packages/aioxmpp/stream.py", line 1661, in _run
    incoming_fut.result())
  File "/home/eloi/dev/push-server/local.virtualenv/lib/python3.5/site-packages/aioxmpp/stream.py", line 957, in _process_incoming
    self._process_incoming_message(stanza_obj)
  File "/home/eloi/dev/push-server/local.virtualenv/lib/python3.5/site-packages/aioxmpp/stream.py", line 811, in _process_incoming_message
    (stanza_obj.type_, stanza_obj.from_.bare()),
AttributeError: 'NoneType' object has no attribute 'bare'
```
Here is [the code I used and the full traceback](https://gist.github.com/azmeuk/767bfcdf7fce217bd7af18f88d99003d).

**Comment by horazont** [comment]
Are you able to test the hotfix/minor-fixes-for-0.7.2 branch? That would be useful in case we find follow-up problems.

**Comment by azmeuk** [comment]
I can confirm this is fixed in the hotfix/minor-fixes-for-0.7.2 branch.

**Comment by horazont** [comment]
@azmeuk Thanks for testing! Have you found any follow-up issues?

**Comment by horazont** [comment]
@azmeuk I released 0.7.2 with the fix. Please check that everything is in working order.

**Comment by azmeuk** [comment]
I just tested with the pip 0.7.2 package. It works fine. Thanks.
